### ISSUE TITLE
Problems with zigbee2mqtt and HUE

### ISSUE THREAD

**Issue by kaelsaes** [Koenkk/zigbee2mqtt#132]
Is it possible that zigbee2mqtt influences my HUE ZigBee network?

When I toggle a few HUE devices with my Xiaomi button sometimes different HUE devices fail to switch on or off.

Switching with a HomeMatic button everthing works fine.

**Comment by Koenkk** [comment]
So you have 2 networks in your house? A hue network and a zigbee2mqtt network. This can cause interference.

**Comment by jarrah31** [comment]
If it helps to give an example of two ZigBee networks working alongside each other, I have a 31 bulb hue network and a 40 item Xiaomi network, with both working side by side without issues for several months.

**Comment by kaelsaes** [comment]
@Koenkk 
What is your recommendation? Should I control my HUE bulbs also with zigbee2mqtt?

**Comment by kaelsaes** [comment]
@jarrah31 
Thanks for your information.
Did you manage the ZigBee channel of both networks?

**Comment by jarrah31** [comment]
Nope, just using them out of the box as such. Don't know much about ZigBee networks and didn't realise there was a potential for them to clash.

**Comment by Koenkk** [comment]
@jarrah31 whats your setup? e.g.

* 31 hue bulbs connected with hue bridge
* 40 xiaomi devices with xiaomi gateway

**Comment by ciotlosm** [comment]
@kaelsaes are you sure it's not a problem with how your automations are setup? I don't see how a change in controller (if there was influence between networks) would cause same devices to behave differently.

**Comment by jarrah31** [comment]
For now yes, but I plan to move all Xiaomi devices onto zigbee2mqtt.

**Comment by jarrah31** [comment]
@kaelsaes Good point - I'll be moving my Xiaomi devices over to Z2M soon so will report back on how it goes.

**Comment by ciotlosm** [comment]
@kaelsaes If you changed which button triggers the lights, it's an extra change besides your gateway change. How is your zigbee stick setup? Does it have `permit_join` true or false?

It is true that two zigbee networks might have some influence to each other but I would expect overcrowding the frequency to be a problem and that is more to do with transmitters than with receiver. 

You can try triggering your lights from home assistant and avoid buttons to check if it has different behaviour.

**Comment by kaelsaes** [comment]
@ciotlosm

Thx

**Comment by ciotlosm** [comment]
@kaelsaes were you able to find the cause of your issue?

**Comment by kaelsaes** [comment]
Not really, but after flashing the C2531 and restarting HUE bridge and zigbee2mqtt it seems to work fine.

**Comment by gohlas** [comment]
Hei  
I know it one year later but a had the same problem this on week ago and while searching for solutions found this topic.
Just to share my solution, what was causing the interference on my HUE hub was that the router channel was set to 11 when I restarted it.
Just changed from auto to channel 6.
After trying different setups I got this one: Zigbee2MQTT channel 11, HUE channel 25 and router channel 6.
So far bit more then one week and no problem at all.
Hope this helps someone in future.
Some information about interference: https://support.metageek.com/hc/en-us/articles/203845040-ZigBee-and-WiFi-Coexistence
